File: LeetCode 75/1732. Find the Highest Altitude.py
# 1732. Find the Highest Altitude

class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        h = 0
        max_h = h
        for i in gain:
            h += i
            max_h = max(max_h, h)
        return max_h

File: LeetCode (en comments)/1732. Find the Highest Altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int high = 0;
        int max_high = high;
        for(int i = 0; i < gain.size(); i++){
            high += gain[i];
            if(high > max_high){
                max_high = high;
            }
        }
        return max_high;
    }
};